sbin/attest-verify

```diff
@@ -11,8 +12,13 @@ import subprocess
 # hard code the hashing algorithm used
 alg = 'sha256'
 
-# attestation directory path
-db_path = 'build/attest'
+# attestation directory path (XXX make configurable)
+db_path = os.environ.get('SAFEBOOT_DB_DIR','build/attest')
+
+tofu_pcrs = []
+if os.path.exists(os.path.join(db_path, 'tofu_pcrs')):
```

Comment:
If the PCRs are tofu'ed, do they end up getting sync'ed at some point?

Reply:
This requires a conversation.  If we're going to use golden PCRs, then how do we decide what they are, especially if there's any reason that they can vary across the plant.  And when firmware gets updated, we need a way to update the golden PCRs.

My preference would be to always have a list of approved firmwares, initramfs and kernel hashes, and allow any PCR 0 and 1 values when they result from extending only with approved values using the eventlog.  But that may not work for all users.

Now, with @geoffthorpe's architecture the attestation servers have a read-only attestation database, so any golden PCR updates would have to go to the enrollment server(s).  And presumably golden PCR updates would be either manual or involve an API invoked by whatever scripts update firmwares.